src/components/GenericDropdown.tsx

```diff
@@ -1,31 +1,28 @@
-import React, { useRef, useState, useEffect } from 'react';
+import React, { useRef } from 'react';
 import styled from 'styled-components';
 import useComponentVisible from '../hooks/useComponentVisible';
 import { Link } from 'react-router-dom';
 
 interface Props {
 	buttonDisplay: React.ReactNode;
+	className?: string;
 }
 
 const DropdownButton = styled.button`
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
-	border: 1px, solid, black;
 `;
 
-const DropdownMenu = styled.ul<{ componentVisible: boolean }>`
+const DropdownMenu = styled.div<{ componentVisible: boolean }>`
 	list-style: none;
 	background-color: ${({ theme }) => theme.colors.main[0]};
 	text-align: start;
 	z-index: 2;
-	position: absolute;
 	border: 1px solid rgba(0, 0, 0, 0.04);
 	box-shadow: 0 16px 24px 2px rgba(0, 0, 0, 0.14);
-	padding: 0;
-	margin: 0;
 	border-radius: 10px;
-	border-radius: 0 0 10px 10px;
+	border-radius: 10px 10px 10px 10px;
```

Comment:
The shorthand will take one argument and apply it everywhere

```suggestion
	border-radius: 10px;
```